.gitignore

```diff
@@ -1,5 +1,6 @@
 MANIFEST
 docs/_build
+borg/
```

Comment:
What's that?

Reply:
That is a result of me mistakenly running `pip install -r requirements.d/docs.txt` in the borg-import directory

Reply:
for development, it is usually nice to have a virtual env outside of the project root.

i usually create a virtual env named projectname-env right besides (not inside) the projectname root dir.

Reply:
Updated my reply: I'll delete my local borgimport/borg/ and remove the line from the .gitignore.

Reply:
don't see that yet in the PR.